### PR TITLE
add postgres metric

### PR DESCRIPTION
**Pull request by ldpliu**
pg_ex-1
pg-ex-2

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/pg_exporter/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            valueFrom:
+              secretKeyRef:
+                name: prometheus-postgres-exporter
+                key: data_source_admin_user
+          - name: DATA_SOURCE_PASS
+            valueFrom:
+              secretKeyRef:
+                name: prometheus-postgres-exporter
+                key: data_source_password
+          image: "quay.io/prometheuscommunity/postgres-exporter:v0.15.0"
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/pg_exporter/deployment.yaml:44]: Do we have this image in Red Hat Downstream?

Comment (ldpliu) [reply operator/pkg/controllers/hubofhubs/manifests/pg_exporter/deployment.yaml:44]: I am not sure if it a downstream 
https://quay.io/repository/prometheuscommunity/postgres-exporter?tab=tags

File: operator/pkg/controllers/hubofhubs/manifests/pg_exporter/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/pg_exporter/deployment.yaml:2]: Can it be a sidecar of postgres sts?

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**